plugins/hazelcast39/src/main/resources/dist-sync.xml

```diff
@@ -10,32 +10,19 @@
     </group>
 
     <network>
-        <port auto-increment="true">5701</port>
-        <outbound-ports>
-            <ports>0</ports>
-        </outbound-ports>
+        <port port-count="200" auto-increment="true">5701</port>
         <join>
-            <multicast enabled="true">
-                <multicast-group>224.0.2.128</multicast-group>
-                <multicast-port>54327</multicast-port>
-            </multicast>
+            <multicast enabled="false"/>
+            <tcp-ip enabled="true">
+                <member>10.0.0.178</member>
+                <member>10.0.0.114</member>
+                <member>10.0.0.126</member>
+            </tcp-ip>
         </join>
     </network>
 
-    <executor-service>
-        <pool-size>16</pool-size>
-        <queue-capacity>0</queue-capacity>
-    </executor-service>
-
     <map name="default">
-        <backup-count>1</backup-count>
```

Comment:
Can you say why you're removing these configuration options?

Reply:
My guess is since it's the default. We used to keep even the defaults to make obvious that this is 1:1 to our `numOwners="2"`. No strong feelings about the removal.